### DISCUSSION TITLE
월드 내에서 누적 방문자 수를 카운팅 할 수 있나요?

### DISCUSSION THREAD

**Discussion by yoon-hw22** [naverz/zepeto-studio-kor#1077] [5. Q&A]
월드 내에서 누적 방문자 수를 카운팅할 수 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이를 설정하신 후 datastorage를 사용해서 카운팅할 수 있습니다.
멀티플레이 가이드 영상 아래 링크 부분을 참고하여 주세요.
https://youtu.be/tfkY_raboV0?t=348

**Reply by yoon-hw22** [reply]
정보를 받아서 월드 내부 게시판 형태로 유저들에게도 제공할 수 있나요?

**Reply by Zepeto-TechEvangelist** [reply]
넵, 서버 데이터를 클라이언트에 보내서 UI에 적용해 주시면 됩니다.
